packages/python-packages/apiview-stub-generator/apistub/_generated/treestyle/parser/models/_patch.py

```diff
@@ -160,13 +160,15 @@ def create_review_line(
         children: Optional[List["ReviewLine"]] = None,
         is_context_end_line: Optional[bool] = False,
         related_to_line: Optional[str] = None,
+        is_handwritten: bool = False,
```

Comment:
Is this a property in model defined in TypeSpec? I assume so since you want to send this is in the json content. Can you please include updated TypeSpec?

Reply:
I actually didn't add the is_handwritten to the CodeModel. It's just used in the Python parser to decide which tokens will have RenderClasses include handwritten.